examples/layer-browser/src/index.js

```diff
@@ -0,0 +1,41 @@
+/* global document */
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import App from './app';
+import FPSStats from 'react-stats-zavatta';
```

Comment:
I generally put the "vendor" dependencies before the app ones, maybe we could discuss on what we want to do for ordering?